engine/flink/management/dev-model/src/main/scala/pl/touk/nussknacker/engine/management/sample/DevProcessConfigCreator.scala

```diff
@@ -105,14 +104,6 @@ class DevProcessConfigCreator extends ProcessConfigCreator {
           processObjectDependencies,
           FlinkKafkaAvroSinkImplFactory
         )
-      ),
-      "kafka" -> all(
```

Comment:
Is it redundant?

Reply:
Before this change, duplicates were checked only across `ComponentProvider`s. Components from ConfigCreators wasn't considered. It caused the silent shadowing of components. I fixed that, some tests shown me that we had an invalid configuration so I fixed it. These components are already delivered by `kafka` `ConfigProvider` which is already in `classPath` in configuration.